internal/evaluator/criteria_test.go

```
@@ -1075,3 +1144,4 @@ func TestOriginalComponentName(t *testing.T) {
 		})
 	}
 }
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)